commit 13f00eb4493c217269b76614759e452d8302955e
Author: Paul Eggert <eggert@cs.ucla.edu>
Date:   Thu Mar 31 16:35:29 2016 -0700

    automake: port to Perl 5.22 and later
    
    Without this change, Perl 5.22 complains "Unescaped left brace in
    regex is deprecated" and this is planned to become a hard error in
    Perl 5.26.  See:
    http://search.cpan.org/dist/perl-5.22.0/pod/perldelta.pod#A_literal_%22{%22_should_now_be_escaped_in_a_pattern
    * bin/automake.in (substitute_ac_subst_variables): Escape left brace.

---
 automake.in |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/automake.in
+++ b/automake.in
@@ -4156,7 +4156,7 @@
 sub substitute_ac_subst_variables ($)
 {
   my ($text) = @_;
-  $text =~ s/\${([^ \t=:+{}]+)}/&substitute_ac_subst_variables_worker ($1)/ge;
+  $text =~ s/\$[{]([^ \t=:+{}]+)}/&substitute_ac_subst_variables_worker ($1)/ge;
   return $text;
 }
 
